_posts/2015-05-27-interesting-work-not-interesting-jobs.md

```diff
@@ -0,0 +1,40 @@
+---
+title: Go for interesting work, not 'interesting' jobs
+layout: posts
+author: Nick Reavill
+author_url: https://www.linkedin.com/in/nreavill
+published: true
+location: "San Francisco"
+---
+A couple of years ago a developer friend and I were discussing our ideal engineering jobs. I didn't have a specific idea of what mine would be but I knew that it would sound boring to most people. Less than a year later I found myself at Stitch Fix doing just that and I could not be happier.
+
+There are two general problems with interesting jobs. The first is that what seems interesting from the outside can turn out to be mostly tedious once you're doing it. I was a singer in many bands during my twenties. Playing live is incredible, and writing songs is pretty good (and presumably much better if you are talented), but most of the time you are rehearsing (playing the same song over and over and over again in a foul-smelling room, mostly being ignored by the actual musicians) or recording (mostly sitting on a foul-smelling sofa drinking tea). From what I can tell acting, especially in movies or on TV, has a similarly high sofa quotient.
+
+The second problem is that the perceived interestingness of a particular job is probably somewhat proportional to the number of people who want that job. If lots of people want to do the same job then competition will be intense which will inevitably lead to downward pressure on wages. Once again, see acting where unemployment rates are above 80%. Elsewhere in the film industry, [entry level jobs][runner-job-description], even for post-graduates, can pay minimum wage.
```

Comment:
a few commas before independent clauses will help. "will be intense *[comma]* which..."  and "see acting *[comma]* where"